pkg/cli/zip_cluster_wide.go

```diff
@@ -81,11 +80,26 @@ var debugZipTablesPerCluster = []string{
 	"crdb_internal.cluster_transactions",
 
 	"crdb_internal.jobs",
-	"system.jobs",       // get the raw, restorable jobs records too.
-	"system.descriptor", // descriptors also contain job-like mutation state.
+	// Get the raw, restorable jobs records too.
+	"system.jobs",
+	// As well as the remaining columns (created_by etc).
+	// See comment on customQuery in zip.go about why we also need this.
+	//
+	// TODO(knz): This can be removed (to keep just system.jobs) once
+	// the table retrieval format preserves column structure.
+	"system.jobs_full",
+
+	// Descriptors also contain job-like mutation state.
+	"system.descriptor",
 	"system.namespace",
 	"system.scheduled_jobs",
 
+	// The synthetic SQL CREATE statements for all tables.
+	// Note the "". to collect across all databases.
+	`"".crdb_internal.create_statements`,
+	// Ditto, for CREATE TYPE.
+	`"".crdb_internal.create_type_statements`,
```

Comment:
These are nice! Perhaps add a comment about how they relate to the `doctor recreate` command? That being said I'm not sure how useful this would be here. Food for thought.
Speaking of `doctor recreate` I'm not sure if this falls in the scope of this PR or not but it's also worth considering using the same base64 literals in that command's output instead of hex encodings, if only for the sake of uniformity. Concretely, do a `decode('%s', 'base64')` in
https://github.com/cockroachdb/cockroach/blob/1b38b77dff11e1da0f016af46fb8e3ab18b627d6/pkg/sql/doctor/doctor.go#L310-L312 instead of `decode('%s', 'hex')`.